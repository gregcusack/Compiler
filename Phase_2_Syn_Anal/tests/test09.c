struct pair {
    int x, y;
    struct pair *next;
};

struct pair s1, s2;

int main(void)
{
    struct pair *sp;
    struct pair **spp;

    1 + 2 * 3 - 4 / 5 && 6 % 7 || 8 == 9;

    sp = &s1;
    spp = &sp;
    sp->x = s1.x;
    (*spp)->next->y = s2.y;
}
