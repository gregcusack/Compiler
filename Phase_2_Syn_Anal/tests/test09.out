mul
add
div
sub
rem
and
eql
or
addr
addr
arrow
dot
deref
arrow
arrow
dot
